rx-java/rxgrpc-stub/src/main/java/com/salesforce/rxgrpc/stub/ClientCalls.java

```diff
@@ -28,12 +29,25 @@ private ClientCalls() {
 
     }
 
+    /**
+     * Sets Prefetch size of queue.
+     */
+    public static final CallOptions.Key<Integer> CALL_OPTIONS_PREFETCH =
+        CallOptions.Key.createWithDefault("reactivegrpc.internal.PREFETCH", Integer.valueOf(512));
+
+    /**
+     * Sets Low Tide of prefetch queue.
+     */
+    public static final CallOptions.Key<Integer> CALL_OPTIONS_LOW_TIDE =
+        CallOptions.Key.createWithDefault("reactivegrpc.internal.LOW_TIDE", Integer.valueOf(512 * 2 / 3));
+
```

Comment:
Can you reference the `DEFAULT_CHUNK_SIZE` and `TWO_THIRDS_OF_DEFAULT_CHUNK_SIZE` constants instead of hardcoding 512? You can move the constants if needed.

https://github.com/salesforce/reactive-grpc/blob/master/common/reactive-grpc-common/src/main/java/com/salesforce/reactivegrpc/common/AbstractStreamObserverAndPublisher.java#L67-L68

Reply:
Sure I can. I just was not able to reach to these and did not know if it is intentional protected. Can I make these public?

Reply:
Sure. Since the defaults are being exposed now, it make sense for the constants to be accessible.

Reply:
Done